lib/internal/child_process.js

```diff
@@ -577,7 +577,7 @@ function setupChannel(target, channel) {
         handle: null,
         message: message,
       });
-      return;
+      return this._handleQueue.length < (65536 * 2);
```

Comment:
Sure. If the threshold is 1, I should probably add a test for that if it's possible to induce it without a ton of load or anything.